tests/x11/gnomecase/application_starts_on_login.pm

```diff
@@ -163,7 +163,13 @@ sub run {
     send_key "alt-f4";
 
     handle_relogin;
-    assert_screen 'xterm';
+    assert_screen [qw(xterm xterm-without-focus)];
+    if (match_has_tag 'xterm-without-focus') {
+        record_soft_failure("poo#66099 - Focus not stolen with GNOME 3.36+ https://gitlab.gnome.org/GNOME/mutter/-/issues/1207");
+        # focus it
+        assert_and_click 'xterm-without-focus';
```

Comment:
```suggestion
        click_lastmatch;
```

Reply:
I was not aware of `click_lastmatch`. Thanks!